src/screens/SUE/SueMapViewScreen.tsx

```diff
@@ -63,6 +65,7 @@ export const SueMapViewScreen = ({ route }: { route: any }) => {
             setSelectedPosition(undefined);
           }
         }}
+        onMarkerPress={(id) => setSelectedMarker(id)}
```

Comment:
`onMarkerPress={setSelectedMarker}` should be fine here, right?

Reply:
Yes 🙌

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1154/commits/800969674da23ee9825df39f73ddb943008dbe34